src/events/commandError.ts

```diff
@@ -4,6 +4,7 @@ import { inlineCodeblock } from '@utils/util';
 import { DiscordAPIError, HTTPError, MessageEmbed } from 'discord.js';
 import { Command, Event, KlasaMessage, util } from 'klasa';
 import { Colors } from '@lib/types/constants/Constants';
+import 'node-fetch';
```

Comment:
why this import btw?

Reply:
Remnants from tests (I wanted to check if it exposed AbortError, turns out they don't).

Nice catch though, thanks!